crates/iota-sdk/src/wallet_context.rs

```diff
@@ -57,10 +61,12 @@ impl WalletContext {
         Ok(context)
     }
 
+    /// Get all addresses from the keystore.
     pub fn get_addresses(&self) -> Vec<IotaAddress> {
         self.config.keystore.addresses()
     }
 
+    /// Get the configured [IotaClient].
```

Comment:
Missing backticks?

Reply:
For me it works without them and cargo doc is also not complaining :thinking:

Reply:
https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html shows also both options, not sure what's the difference would be

Reply:
Let's not have a mix of both anyway 😬

Reply:
It's just the formatting that is different

Reply:
Changed and yes found this explanation
```
[std::iter::Iterator], without having a link reference definition for Iterator anywhere else in the document
[`std::iter::Iterator`], without having a link reference definition for Iterator anywhere else in the document (same as previous style but with back ticks to format link as inline code)
```